front/app/containers/Admin/messaging/texting/CampaignList/TextCampaignListRow.tsx

```diff
@@ -14,54 +11,52 @@ import styled from 'styled-components';
 
 // i18n
 import { FormattedTime, FormattedDate } from 'react-intl';
-import { FormattedMessage } from 'utils/cl-intl';
+
+// utils
+import clHistory from 'utils/cl-router/history';
+import { truncate } from 'utils/textUtils';
 
 interface Props {
   campaign: ITextingCampaignData;
 }
 
-const Container = styled.div`
-  width: 100%;
+const Row = styled.tr`
   height: 50px;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
   border-top: 1px solid #e0e0e0;
+  cursor: pointer;
 `;
 
-const Left = styled.div`
-  display: flex;
-  align-items: center;
-  gap: 20px;
-  width: 100%;
+const BodyText = styled.div`
+  margin-right: 20px;
 `;
 
-const TextWrapper = styled.div`
-  line-height: 50px;
+const DateTime = styled.div`
+  text-align: left;
+  white-space: nowrap;
+  margin-right: 20px;
 `;
 
-const Text = styled.p`
-  width: 600px;
+const SentText = styled.div`
+  text-align: left;
   white-space: nowrap;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  font-size: 16px;
+  margin-right: 20px;
 `;
 
-const Right = styled.div`
-  display: flex;
-  gap: 30px;
+const Cell = styled.td`
+  padding-right: 20px;
 `;
 
-// 140px is sufficient for longest date-time string
-const DateTime = styled.div`
-  width: 140px;
+const MessageCell = styled(Cell)`
+  width: 67%;
+  margin-left: 20px;
```

Comment:
I don't think the margin-left is doing anything here, can be removed. 

For font-size, we should use `font-size: ${fontSizes.base}px` instead of `font-size: 16px`.